content/documentation/staging/installation-guide/index.adoc

```diff
@@ -68,10 +68,17 @@ For development the requirements are lower, depending on how demanding your appl
 For more, see the link:https://docs.okd.io/latest/welcome/index.html[OKD install documentation].
 
 
+=== Helm Requirements
+
+If you are going to install the Helm Chart, *you must use Helm 3*. You can install Helm by following the link:https://helm.sh/docs/intro/install/[Helm installation docs].
+
+
 == Installation
 
 Kiali can be installed in an OpenShift or Kubernetes cluster environment. For OpenShift only, a minimal preparation is necessary.
```

Comment:
OpenShift is Kubernetes, no?  Maybe just say,
"If installing Kiali into the OpenShift Kubernetes distribution then follow the minimal preparation below.  For other Kubernetes distributions you can skip to the next section."

Also, move this below the bullhorn and lightbulb.

Also, if this is a valid requirement it would also be needed by the quick start, which users can navigate to directly.

Reply:
done.